homeassistant/components/sleepiq/entity.py

```diff
@@ -65,3 +65,19 @@ def _handle_coordinator_update(self) -> None:
     @abstractmethod
     def _async_update_attrs(self) -> None:
         """Update sensor attributes."""
+
+
+class SleepIQBedCoordinator(CoordinatorEntity):
```

Comment:
It's confusing to name this entity class coordinator. Please replace that word with something else or add `Entity` to the end.